bin/phpcs-requirelist-commands.js

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-process-exit */
+const spawnSync = require( 'child_process' ).spawnSync;
+const chalk = require( 'chalk' );
+const requirelist = require( './phpcs-requirelist' );
+
+requirelist.forEach( function ( fileToProcess ) {
+	if ( 'docker/' === fileToProcess ) {
+		console.log( chalk.yellow( 'Skipping docker/. It takes forever.' ) );
+		return;
+	}
+	console.log( chalk.yellow( 'Processing ' + fileToProcess ) );
+	spawnSync( 'vendor/bin/phpcbf', [ fileToProcess ], {
```

Comment:
No real reason; duplicated the call in the pre-commit hook and since the alias is rather long, opted to use it. But, I didn't go into why the alias exists or why it passes the option it does.